application/dash_application/views/pre_exome.py

```diff
@@ -48,6 +47,28 @@
     'data-table'
 ])
 
+BAMQC_COL = BamQcColumn
+PINERY_COL = pinery.column.SampleProvenanceColumn
+INSTRUMENT_COLS = pinery.column.InstrumentWithModelColumn
+RUN_COLS = pinery.column.RunsColumn
+
+special_cols = {} #TODO 
+
+def get_bamqc_data():
+    bamqc_df = QCETLCache().bamqc.bamqc
+    bamqc_df = util.df_with_normalized_ius_columns(bamqc_df, BAMQC_COL.Run, BAMQC_COL.Lane, BAMQC_COL.Barcodes)
+
+    pinery_samples = util.get_pinery_samples_from_active_projects()
+    # TODO filter??
+
+    bamqc_df = util.df_with_pinery_samples(bamqc_df, pinery_samples, util.bamqc_ius_columns)
+
+    bamqc_df = util.df_with_instrument_model(bamqc_df, PINERY_COL.SequencerRunName)
+
+    return bamqc_df
```

Comment:
I think this will include all WG samples as well. Could you add a `util.filter_by_library_design` call to select just the `EX` and `TS` samples?